google/ggrc-core
# GGRC-6841 Introduce map permission

# Dependencies

This PR is `on hold` until the dependencies are merged:

- [ ] GGRC-6867

# Issue description

There are 4 permissions in the system now:

- Create
- Read
- Update
- Delete

New ‘Map’ permission should be added.

# Steps to test the changes

In scope of this PR all global and object level roles will receive map permission which is the same as update permission. This is needed to provide separate delivery of Map permission for each role in next PRs.

For custom roles: 

1. Create custom roles for any object type;
2. Check that the user with update permission and without map permission can not map object to any other object;
3. Check that the user without update permission and with map permission can map object to any other object.

# Solution description

Added 'map' column to acr table and filled with the same values as 'update'. Updated permission dicts for global roles. Added minor updates to rbac related logic. Updated converters. Updated revisions.

# Sanity checklist




# PR Review checklist
